ouroboros-network/src/Ouroboros/Network/Diffusion/Policies.hs

```diff
@@ -41,6 +42,18 @@ closeConnectionTimeout :: DiffTime
 closeConnectionTimeout = 120
 
 
+-- | Chain sync `mustReplayTimeout` lower bound.
+--
+minChainSyncTimeout :: DiffTime
+minChainSyncTimeout = 135
+
+
+-- | Chain sync `mustReplayTimeout` upper bound.
+--
+maxChainSyncTimeout :: DiffTime
+maxChainSyncTimeout = 269
+
+
```

Comment:
👍 we should update consensus to use these instead of 135 and 269 directly..
The comment in consensus should be moved here, that is the talk about the false positive thresholds. These are not just magical numbers.

Reply:
Done in `coot/ouroboros-network-0.16` branch in `ouroboros-consensus`.